### PR TITLE
fix #1582: [EN] Support 'non-gloss', alias of 'non-gloss definition'

### LINKED ISSUE #1582: [EN] Support 'non-gloss', alias of 'non-gloss definition'
- Wiktionary page: https://en.wiktionary.org/wiki/Eternal_City

Wikicode:
```
{{non-gloss|A [[nickname]] for the city of [[Rome]].}}
```

Output:
```
Non-gloss
```

Expected:
```
<i>A [[nickname]] for the city of [[Rome]].</i>
```

---

Model link, if any: https://en.wiktionary.org/wiki/Template:non-gloss_definition

### PR DESCRIPTION
**Pull request by lasconic**
Fixes #1582

### CODE REVIEW

File: wikidict/lang/en/__init__.py [hunk 1]
```diff
@@ -47,7 +47,10 @@
     "en-past of",
     "en-simple past of",
     "en-superlative of",
+    "en-tpso",
```
Comment (BoboTiG) [inline wikidict/lang/en/__init__.py:50]: Might be good to add that same value to https://github.com/BoboTiG/ebook-reader-dict/blob/fd31b20cf26c945ce1079e220ac5a9241789e612/wikidict/render.py#L387

File: wikidict/lang/en/__init__.py [hunk 2]
```diff
@@ -148,14 +168,19 @@
     "en-simple past of": "parts[1]",
     # {{en-irregular plural of|term}}
     "en-irregular plural of": "parts[1]",
+    "en-ipl": "parts[1]",
```
Comment (BoboTiG) [inline wikidict/lang/en/__init__.py:171]: Same here.

File: wikidict/svg.py [hunk 3]
```diff
@@ -25,7 +25,7 @@ def get(formula: str) -> str:
 
 
 def set(formula: str, svg_raw: str) -> None:
-    print(f"{formula!r}: {svg_raw!r},")
+    # print(f"{formula!r}: {svg_raw!r},")
```
Comment (BoboTiG) [inline wikidict/svg.py:28]: Is there an issue?

Comment (lasconic) [reply wikidict/svg.py:28]: it was a bit verbose when I was debugging. I will put it back.

File: wikidict/lang/en/__init__.py (hunk 2 above)
Comment (lasconic) [reply wikidict/lang/en/__init__.py:171]: ok.

File: wikidict/lang/en/__init__.py (hunk 1 above)
Comment (lasconic) [reply wikidict/lang/en/__init__.py:50]: ok

File: wikidict/svg.py (hunk 3 above)
Comment (BoboTiG) [reply wikidict/svg.py]: If that prints something, then we are missing some SVGs in our cache. Just copy-paste the lines, and run `python sort-svg-cache.py` (+ commit the changes).

Comment (lasconic) [reply wikidict/svg.py]: mmm, now I need to remember when it printed :)

**Pull request merged**